src/esp/bindings/SimBindings.cpp

```diff
@@ -133,7 +146,7 @@ void initSimBindings(py::module& m) {
       .def("set_object_bb_draw", &Simulator::setObjectBBDraw, "draw_bb"_a,
            "object_id"_a, "sceneID"_a = 0)
       .def("recompute_navmesh", &Simulator::recomputeNavMesh, "pathfinder"_a,
-           "navmesh_settings"_a, "include_static_objects"_a)
+           "navmesh_settings"_a, "include_static_objects"_a = true)
```

Comment:
It was `false originally`
```suggestion
           "navmesh_settings"_a, "include_static_objects"_a = false)
```